### PR TITLE
remove appeals api feature flag

### PR DESCRIPTION
**Pull request by charleystran**
The PR removes the feature flag so that the Appeals API tab and Decision Reviews sub section appear on the production developer portal.

### CODE REVIEW

**Comment by rtravitz** [comment]
Are the diffs okay on the snapshots? Can approve if they're as expected and they get updated.

**Comment by charleystran** [comment]
@mjlumetta correct because they are internal only. for the typo issue, I can fix in a follow up PR where we move the current appeals status from the benefits tab to the appeals tab. If you are ok with that

File: src/content/apiDocs/appeals/appealsReleaseNotes.mdx [hunk 1]
```diff
@@ -1,2 +1,2 @@
-### Month 99, 2099
-(v1) These are dummy release notes.  Link to PRs here!
+### June 06, 2020
```
Comment (mjlumetta) [inline src/content/apiDocs/appeals/appealsReleaseNotes.mdx:1]: should this be "6" instead of "06"? (not a blocking question for me)

Comment (charleystran) [reply src/content/apiDocs/appeals/appealsReleaseNotes.mdx:1]: just following the 99, but I can change it if you want. Not a huge deal. I would defer to your teams opinion as you are the owners of the portal

**Pull request merged**